examples/http-external/index.ts

```diff
@@ -0,0 +1,15 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as azure from "@pulumi/azure";
+
+const resourceGroup = new azure.core.ResourceGroup('example', { location: 'West US 2' });
+
+// Create a Function App implemented in PowerShell with source code from 'funcapp' folder
+const app = new azure.appservice.MultiFunctionApp("http-ps", {
+    resourceGroupName: resourceGroup.name,
+    archive: new pulumi.asset.FileArchive("./funcapp"),
+    appSettings: {
+      "runtime": "powershell",
+    }
+});
+
+export const url = app.endpoints.apply(ep => `${ep[0]}HelloPS?name=Pulumi`);
```

Comment:
not quite clear to me why a MultiFunctionApp (with no additional args) would end up with an endpoint.  Is that normal behavior?

Reply:
Every Function App has a default endpoint like `https://bla.azurewebsites.net/api/`. If no functions exist in the app, the endpoint will return 404. Functions can handle either `https://bla.azurewebsites.net/api/MyFunctionName` or any route if the route pattern is specified.

Even for an archive-based app, we should return an endpoint back for convenience.